lib/fake_async.dart

```diff
@@ -272,13 +276,23 @@ class _FakeTimer implements Timer {
   /// fired.
   Duration _nextCall;
 
+  /// The current stack trace when this timer was created.
+  final StackTrace _creationStackTrace;
+
   @override
   int get tick {
     throw UnimplementedError('tick');
   }
 
+  /// Returns debugging information to try to identify the source of the
```

Comment:
can you expand this and say what information it includes?

Reply:
done